.github/issue_template.md

```diff
@@ -0,0 +1,23 @@
+<!-- GitHub is for bugs, feature and enhancement requests, and discussion on developing Service Fabric, as opposed to using it.
+If you need help or have any questions regarding using Service Fabric, you can ask on [Stack Overflow](https://stackoverflow.com/questions/tagged/azure-service-fabric)
+-->
+> If you know the area of code this belongs to, please select an appropriate tag.
+> Otherwise, select `triage-needed`
+
+**Bug or Enhancement:**
```

Comment:
or question

Reply:
The point of this question was to discourage product questions by explicitly making the user choose between the things we DO want on the Github: bugs and enhancements.  The idea is that if they have a product question, and they go to file an issue, they get to the "Bug or Enhancement" question, and they don't have a valid answer ("neither") so they instead get redirected to the above StackOverflow